app/assets/javascripts/discourse/tests/acceptance/search-test.js

```diff
@@ -3,22 +3,28 @@ import {
   count,
   exists,
   query,
-  queryAll,
 } from "discourse/tests/helpers/qunit-helpers";
 import { click, fillIn, triggerKeyEvent, visit } from "@ember/test-helpers";
+import I18n from "I18n";
 import searchFixtures from "discourse/tests/fixtures/search-fixtures";
 import selectKit from "discourse/tests/helpers/select-kit-helper";
 import { test } from "qunit";
 
 acceptance("Search - Anonymous", function (needs) {
-  let calledEmpty = false;
-
   needs.pretender((server, helper) => {
     server.get("/search/query", (request) => {
-      if (!request.queryParams["search_context[type]"]) {
-        calledEmpty = true;
+      if (request.queryParams.type_filter === "exclude_topics") {
+        return helper.response({
+          posts: [],
+          topics: [],
```

Comment:
When excluding topics, are the `posts` and `topics` attributes included in the payload?